easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-20.10-iimpi-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,61 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'CMakeMake'
+name = 'OpenMolcas'
+version = '20.10'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://gitlab.com/Molcas/OpenMolcas"
+description = "OpenMolcas is a quantum chemistry software package"
+
+toolchain = {'name': 'iimpi', 'version': '2020a'}
+
+source_urls = ['https://gitlab.com/Molcas/OpenMolcas/-/archive/v%(version)s/']
+sources = ["OpenMolcas-v%(version)s.tar.gz"]
+
+# This patch avoids a forced extra copy of 'pymolcas' script
+# in a local folder of the user doing the build
+# The 'make install' process will already install it in
+# %(installdir)s/sbin/
+patches = ["OpenMolcas-pymolcas_only_in_ebprefix.patch"]
+
+checksums = [
+    # OpenMolcas-20.10.tar.gz
+    '73f73beaebbcff5d02b5bb0748d07c5b8e81784fae6122729c7e7aeed730fe42',
+    # OpenMolcas-pymolcas_only_in_ebprefix.patch
+    '56165ab4a752ef7cdbe3340c8c922fca04510e8ee2e4024ea654e6bcd83198f8'
+]
+
+dependencies = [
+    ('GlobalArrays', '5.8'),
+    ('HDF5', '1.12.0'),
```

Comment:
@nordmoen Please stick to HDF5 1.10.6 in the `2020a` generation of easyconfigs, to ensure these easyconfigs are compatible with others of the same generation.